components/script/dom/webglrenderingcontext.rs

```diff
@@ -644,6 +647,12 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
             }
         } else {
             constants::NO_ERROR
+            //Todo(Mortimer): Enable GPU getError() when the webtraits repo is merged
+            //let (sender, receiver) = ipc::channel().unwrap();
+            //self.ipc_renderer
```

Comment:
Right now they *are* run, which is a nice perf impact, but I plan to make them not run once https://github.com/servo/servo/pull/13387 is landed.